specs/stork-daemonset.yaml

```diff
@@ -103,7 +58,7 @@ subjects:
   namespace: kube-system
 roleRef:
   kind: ClusterRole
-  name: stork-role
+  name: cluster-admin
```

Comment:
we want to make stork cluster admin?

Reply:
For migrating it needs to look at all resources, might as well make it an admin

Reply:
This is how a lot of system level services do it (helm, ark, to name a new). We already have so many permissions here which is borderline cluster admin anyways.